Makefile

```diff
@@ -162,8 +162,8 @@ check: compile-cairo compile-starknet compile-cairo-1-casm compile-cairo-1-sierr
 deps: check-python-version build-cairo-2-compiler build-cairo-1-compiler
 	cargo install flamegraph --version 0.6.2
 	cargo install cargo-llvm-cov --version 0.5.14
-	-pyenv && pyenv install -s pypy3.9-7.3.9
```

Comment:
why did you removed this part?

Reply:
I think it was giving me problems.
The `pyenv` command without anything seems to be equivalent to `pyenv --help`, so it wasn't doing anything anyways.
Should I restore it anyways?

Reply:
I think that's done like that so if you don't have `pyenv` it fails silently and assumes you have a "good" Python version, but if it fails for any other reason you still find out. Does that make sense?

Reply:
Yes, but I think that it doesn't change the fact that it's unnecessary. As far as I'm aware, the `-` symbol on front of the line is the one responsible for that behaviour, not the `pyenv &&`.